summary_scores_util.py

```diff
@@ -24,13 +24,13 @@ def __init__(self, module_init_script):
       self.output_form[i] = module.output_form
 
   # dataframe and errorFlag
-  def compute_scores(self, instrument, input_data, demographics):
-    scoresDF = self.functions[instrument](input_data, demographics) 
+  def compute_scores(self, instrument, input_data, demographics, log=None, **kwargs):
+    scoresDF = self.functions[instrument](input_data, demographics, log=log, **kwargs)
```

Comment:
So my only concern here is that `log` could be `None`; and hence each of the scoring scripts would have to test to see if `log` is `None` before using. 

We should either create some default logger that either goes to stdout/stderr or does nothing. Otherwise my suggestion as we discussed in Slack to hook python logging into slog somehow; then each script could explicitly grab the appropriate logger instance that automatically gets wired up to sibislogger.

Reply:
Fixed in 5fdcaea.